### ISSUE TITLE
This project code is lost. Time, Token and Money is lost. Can you get this fixed?

### ISSUE THREAD

**Issue by jpmirrar** [stackblitz/bolt.new#1802]
### Describe the bug

I have created this project but the code is not available after reloading the project. If you are not able to restore the project. Need the tokens to be refunded.  

### Link to the Bolt URL that caused the error

https://bolt.new/~/bolt-vite-react-tnspc3

### Steps to reproduce

Load the project. 

### Expected behavior

Loading the project should load all the code that was generated.

![image](https://github.com/user-attachments/assets/08d48f52-2c24-48f0-96ca-6d6d008acb24)

### Screen Recording / Screenshot

![image](https://github.com/user-attachments/assets/7608878f-5bfd-4509-b14a-ac5b5238cc62)

### Platform

```
Browser name = Chrome
Full version = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36 Edg/130.0.0.0
performance.memory = {
  "totalJSHeapSize": 117020089,
  "usedJSHeapSize": 113010765,
  "jsHeapSizeLimit": 4294705152
}
Username = jpmirrar
Chat ID = 36d1e0c231b1
```

**Comment by endocytosis** [comment]
Hi @jpmirrar, 

Appreciate the feedback here! We are aware of this issue with chat persistence. Temporary workarounds and updates can be found here: #39. You can go to stackblitz.com, login with your Bolt credentials, click Collections on the left-hand side, and then open Bolt Collection. Appreciate your patience as improvements continue to be made. Refund requests are handled by reaching out to hello@stackblitz.com

**Comment by jpmirrar** [comment]
@endocytosis The issue for me is not about the chat but the code itself is lost, it is not available in StackBlitz. I only have the boilerplate code generated but the code that was created using bolt.new is completely lost.

This happened when I selected the React as a choice to create the project in bolt.new. Whereas when I created any project without selecting the templates available in the bottom of the page then it is working as expected. 

This issue is not the same as #39.
